packages/core/src/index.html

```diff
@@ -61,6 +61,14 @@
 
     <noscript>Your browser does not support JavaScript!</noscript>
 
+    <!-- Start Segment Script -->
+    <script type="text/javascript">
```

Comment:
Can we add this tag via GTM? :thinking:

Reply:
AFAIK, these two are different things, and we can put GTM under segment in the later phase. @ashkanx ?

Reply:
@easteregg @mustofa-binary IIRIC, Segment doesn't recommend this. I prefer to add it separately too. But I'll ask segment team again

Reply:
@easteregg @mustofa-binary please check out [here](https://segment.com/docs/guides/general/what-is-the-difference-between-segment-and-tag-managers/)